go/service/config.go

```diff
@@ -516,3 +516,15 @@ func (h ConfigHandler) AppendGUILogs(ctx context.Context, content string) error
 	_, err := io.WriteString(wr, content)
 	return err
 }
+
+func (h ConfigHandler) GenerateWebAuthToken(ctx context.Context) (ret string, err error) {
+	nist, err := h.G().ActiveDevice.NISTWebToken(ctx)
+	if err != nil {
+		return ret, err
+	}
+	if nist == nil {
+		return ret, fmt.Errorf("cannot generate a token when you are logged off")
+	}
+	uri := libkb.SiteURILookup[h.G().Env.GetRunMode()] + libkb.APIURIPathPrefix + "/user/token.json?tok=" + nist.Token().StringURL()
```

Comment:
I would probably go with different endpoint name, but otherwise LGTM

Reply:
Sure, what do you think it should be? I already changed it to be non-API